docs/ref/forms/api.txt

```diff
@@ -934,6 +934,30 @@ When set to ``True`` (the default), required form fields will have the
 ``use_required_attribute=False`` to avoid incorrect browser validation when
 adding and deleting forms from a formset.
 
+.. attribute:: Form.bound_field_class
```

Comment:
This addition is inside a subsection named "Configuring form elements' HTML ``id`` attributes and ``<label>`` tags". I'm not sure this content is relevant for that? Could you please share your rationale for why this was added here? Thanks.

Reply:
Maybe a good location would be inside the "Styling required or erroneous form rows"?

Reply:
Mentioning it in the form rows section is a great idea; I added a link from there to the `Customizing BoundField` section.